docs/releasenotes/release-1.125.md

```diff
@@ -1,53 +1,43 @@
-    the Direct Cloud Reconciler, which fixes issues with updating from ENTERPRISE -> ENTERPRISE_PLUS edition and allows
-    "create from clone" functionality.
-
-## New Resources:
+  * You can use the `alpha.cnrm.cloud.google.com/reconciler: direct` annotation on the `SQLInstance` CR object to opt-in the direct controller, which fixes issues with updating from ENTERPRISE -> ENTERPRISE_PLUS edition and allows "create from clone" functionality.
 
-* Added support for `PlaceholderKind` (v1beta1) resource.
-
-## New Fields:
+* `ComputeFirewallPolicyRule`
 
-* `AlloyDBInstance`
-  * Added `spec.networkConfig.enableOutboundPublicIp` field.
-  * Added `status.outboundPublicIpAddresses` field.
+  * You can use the `alpha.cnrm.cloud.google.com/reconciler: direct` annotation on the `ComputeFirewallPolicyRule` CR object to opt-in the direct controller, which fixes the `targetResources` error *required value "priority" could not be found*.
 
 ## New features:
 
-* Allow more customization of resource reconciliation in cluster mode
-  * Added a new `ControllerReconciler` CRD (v1alpha1). See [example](https://github.com/GoogleCloudPlatform/k8s-config-connector/blob/master/operator/config/samples/controller_reconciler_customization_sample.yaml)
-  * This feature allows users to customize the client-side kube-apiserver request rate limit when Config Connector is runnning in cluster mode.
+* Add *cluster mode* to manage the rate-limit for the Config Connector requests
 
-## Bug Fixes:
+  * Previously we added [rate-limit control in namespace mode](https://cloud.google.com/config-connector/docs/how-to/customize-controller-manager-rate-limit). Users can configure the `ControllerReconciler` object to set the rate-limit for the reconciling requests to the kube-apiserver for their Config Connector resources. 
+  * This release adds this feature in the cluster mode. This [example](https://github.com/GoogleCloudPlatform/k8s-config-connector/blob/master/operator/config/samples/controller_reconciler_customization_sample.yaml) shows how to set up the configuration.
 
-* [Incorrect format of clientTLSPolicy when referenced from ComputeBackendService](https://github.com/GoogleCloudPlatform/k8s-config-connector/pull/3007)
+## Bug Fixes:
 
-* [Fix](https://github.com/GoogleCloudPlatform/k8s-config-connector/pull/2973) the reconciliation error in `ContainerNodePool` that occurs when `kubeletConfig` is empty in the user's configuration, but a subfield under `kubeletConfig` is set externally outside of KCC.
+* [Issue 3007](https://github.com/GoogleCloudPlatform/k8s-config-connector/pull/3007) ComputeBackendService cannot refer clientTLSPolicy due to invalid format
+* [Issue 2973](https://github.com/GoogleCloudPlatform/k8s-config-connector/pull/2973) kubelet_config has `insecure_kubelet_readonly_port_enabled: true` set even if not configured in the `ContainerNodePool` object. 
```

Comment:
I don't feel this accurately describe the issue we resolved. The `insecure_kubelet_readonly_port_enabled` field will be set to `true` because of underlying API changes (from GCP). Our code change addresses the constant reconciliation error on the `ContainerNodePool` resource triggered by this API change.

Reply:
Release note is for KCC users, so we need to explain the problem in the way that user can understand and only talk about the things users care.